br/.golangci.yml

```diff
@@ -0,0 +1,41 @@
+run:
```

Comment:
Can we reuse the `.golangci.yml` in root directory ?

Reply:
should reuse .golangci.yml in root directory